js/src/tooltip.js

```diff
@@ -747,9 +747,8 @@ class Tooltip {
   _cleanTipClass() {
     const tip = this.getTipElement()
     const tabClass = tip.getAttribute('class').match(BSCLS_PREFIX_REGEX)
-    if (tabClass !== null && tabClass.length) {
-      tabClass
-        .map(token => token.trim())
+    if (tabClass !== null && tabClass.length > 0) {
```

Comment:
`tabClass.length` is equal too `tabClass.length > 0` not really needed that change

Reply:
It's what we do in poper.js; this code is exactly the same

Reply:
So we can change that in `popover.js` too 😄 but it's nitpicking

Reply:
Fine by me, I'd just prefer if we enforced this with a rule, but I cannot find one, only for the opposite `unicorn/explicit-length-check`